docs/communicate-with-javascript.md

```diff
@@ -550,6 +550,71 @@ let () = match [%bs.external __filename] with
 | None -> Js.log "non-node environment"
 ```
 
+## Inlining constant values
+
+Some JavaScript idioms require special constants to be inlined since they serve
+as de-facto directives for bundlers. A common example is `process.env.NODE_ENV`:
+
+```js
+if(process.env.NODE_ENV !== 'production') {
+   // Development-only code
+}
+```
+
+becomes:
+
+```js
+if('development' !== 'production') {
+   // Development-only code
+}
+```
+
+In this case, bundlers such as Webpack can tell that the `if` statement always
+evaluates to a specific branch and eliminate the others.
+
+Melange provides the `[@bs.inline]` construct to achieve the same goal in
+generated JavaScript. Let's look at an example:
```

Comment:
```suggestion
generated JavaScript. Let’s look at an example:
```

https://english.stackexchange.com/a/36048

Reply:
I think it's Option + Shift + ] on macos us keyboards. In any case, I can make a find/replace when we are done with v1.